lib/interactive.py

```diff
@@ -250,25 +243,16 @@ def plot_route(self, messageId):
                             else:
                                 msgType = "Forwarding message"
 
-                    fields = []
-                    msgTypeField = r"$\bf{"+msgType+"}$"
-                    fields.append(msgTypeField)
-                    origSenderField = "Original sender: "+str(p.packet["from"]-HW_ID_OFFSET)
-                    fields.append(origSenderField)
-                    destField = "Destination: "+to
-                    fields.append(destField)
-                    portNumField = "Portnum: "+str(p.packet["decoded"]["simulator"]["portnum"])
-                    fields.append(portNumField)
-                    if hopLimit:
-                        hopLimitField = "HopLimit: "+str(hopLimit)
-                        fields.append(hopLimitField)
-                    rssiField = "RSSI: "+str(round(p.rssis[ri], 2)) + " dBm"
-                    fields.append(rssiField)
-                    table = ""
-                    for i, f in enumerate(fields):
-                        table += f
-                        if i != len(fields)-1:
-                            table += "\n"
+                    hopLimit = p.packet.get("hopLimit")
+
+                    fields = [ r"$\bf{" + msgType + "}$"
+                             , f"Original sender: {p.packet["from"] - HW_ID_OFFSET}"
```

Comment:
I get a syntax error here:
`SyntaxError: f-string: unmatched '['`, probably need to replace the first double quotes with a single quote.